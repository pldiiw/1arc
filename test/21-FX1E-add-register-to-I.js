'use strict';

let engine = require('engine').initialize();
const addRegisterToI = require('instructions').addRegisterToI;

engine.data[0xA] = 50;
engine.I = 4000;

const engine1 = addRegisterToI(engine, 0xA);
const engine2 = addRegisterToI(engine1, 0xA);

console.log(engine.I);
console.log(engine1.I);
console.log(engine2.I);
