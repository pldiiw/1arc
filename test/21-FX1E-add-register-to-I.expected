4000
4050
4
